tests/unit/ecs_tasks/test_json.py

```diff
@@ -30,25 +30,6 @@ def test_it_generates_new_json_file_without_matches():
     )
 
 
-def test_it_handles_json_with_gzip_compression():
```

Comment:
I felt it was a redundant test because the `delete_matches_from_json_file` never receives a gzipped json file now as the unzipping happens earlier with pyarrow. If the method receives a gzipped file it will fail because now it doesn't handle compression at all.

I don't think it is necessary to test a gzipped file at all in this function since the `execute` function will have unzipped and will only pass a bytestream down to `delete_matches_from_json_file` with my changes.

Reply:
Ok I think you're right, it makes sense.